### PR TITLE
Add support for statement version property

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

**Comment by Lctrs** [comment]
`php-xapi/json-statement-fixtures` should be updated too.

File: src/Normalizer/StatementNormalizer.php [hunk 1]
```diff
@@ -80,6 +82,16 @@ public function supportsNormalization($data, $format = null)
      */
     public function denormalize($data, $class, $format = null, array $context = array())
     {
+        $version = null;
+
+        if (isset($data['version'])) {
+            $version = $data['version'];
+
+            if (!preg_match('/^1\.0(?:\.\d+)?$/', $version)) {
```
Comment (xabbuh) [inline src/Normalizer/StatementNormalizer.php:93]: The version returned by an LRS must always have the patch part, doesn't it?

Comment (Lctrs) [reply src/Normalizer/StatementNormalizer.php:93]: Yes. But here, we deserialize Statement that come from the Clients, which means the version can also be in the form of `1.0` according to the [spec](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#version).

> Version MUST be formatted as laid out for the API version header in [Versioning](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#versioning).

**Comment by xabbuh** [comment]
Thank you @Lctrs.

**Pull request merged**